Resources/Prototypes/_NF/Roles/Jobs/Pirates/pirate_first_mate.yml

```
@@ -5,11 +5,11 @@
   playTimeTracker: JobPirateFirstMate
   requirements:
     - !type:OverallPlaytimeRequirement
-      time: 21600 # 6 hours # mono
-    - !type:RoleTimeRequirement # mono
-      role: JobPirate # mono
-      time: 10800 # 3 hours
-  whitelisted: true # mono - comment out
+      time: 64800 # 18 hours # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)